src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(health = 100, name) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  ifDead() {
```

Comment:
The method name `ifDead` is not very descriptive. Consider renaming it to something like `checkIfDead` to better convey its purpose.